### PR TITLE
Fix codenarc rule: naming

### PR DESCRIPTION
**Pull request by zpencer**
Also fix a 120 character issue that slipped into the repo
Disable two naming rules because we already break them a lot in our repo and it's not worthwhile to fix them all:
- our test cases have spaces in their method names
- some of our method names are the same names as internal fields

### CODE REVIEW

File: src/main/groovy/com/google/protobuf/gradle/ProtobufPlugin.groovy [hunk 1]
```diff
@@ -251,13 +251,21 @@ class ProtobufPlugin implements Plugin<Project> {
...
         // For Android Gradle plugin >= 2.5
         Attribute artifactType = Attribute.of("artifactType", String)
-        Task extractIncludeProtosTask =
-            maybeAddExtractIncludeProtosTask(
-                    variant.name,
-                    variant.compileConfiguration.incoming.artifactView{ attributes{ it.attribute(artifactType, "jar") }}.files,
-                    variant.hasProperty("testedVariant") ?
-                            variant.testedVariant.compileConfiguration.incoming.artifactView{ attributes{ it.attribute(artifactType, "jar") }}.files :
-                            null)
+        String name = variant.name
```
Comment (zhangkun83) [inline src/main/groovy/com/google/protobuf/gradle/ProtobufPlugin.groovy]: `variant.name` was only used once. Why is this needed?

Comment (zpencer) [inline src/main/groovy/com/google/protobuf/gradle/ProtobufPlugin.groovy]: My intellij was complaining that the types could not be inferred, and explicitly assigning the vars this way made it quiet
